src/mscorlib/shared/System/IO/FileStream.Win32.cs

```diff
@@ -3,14 +3,29 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Win32.SafeHandles;
-using System.Runtime.InteropServices;
 
 namespace System.IO
 {
     public partial class FileStream : Stream
     {
+        private static bool IsFromAppAvailable()
```

Comment:
Also, this logic may rather be in CoreCLR specific file. We do not need the dynamic switching at runtime either in CoreRT console apps.